gh-codeql

```diff
@@ -92,10 +92,23 @@ if [ "$1" = "set-channel" ]; then
     exit 0
 fi
 
+
+function update_latest {
+    if [ "$channel" = "release" ]; then
+        # For the release channel, we sort by semantic version order since the tags correspond to release versions.
+        # We ignore draft releases and pre-release versions in this case since we want to give the user a stable version.
+        latest=$(gh api "repos/$repo/releases" --paginate --jq ".[] | select(.draft == false and .prerelease == false) | .tag_name" | sort -V | tail -1)
```

Comment:
Would it be nicer to use a style where `update_latest` just prints a single line to stdout, and is called as `"$(update_latest)"`, instead of communicating via global variables?

Reply:
Agreed, that looks much nicer.

Reply:
I guess it would be better named `get_latest_version` in that style...

Reply:
Indeed, I also renamed it to `get_latest` when I did that 🙂